tests/ui/async-await/no-unsafe-async.stderr

```diff
@@ -1,16 +1,11 @@
 error: expected one of `extern` or `fn`, found keyword `async`
   --> $DIR/no-unsafe-async.rs:7:12
    |
-LL | impl S {
-   |        - while parsing this item list starting here
-LL |     #[cfg(FALSE)]
 LL |     unsafe async fn g() {}
    |     -------^^^^^
    |     |      |
    |     |      expected one of `extern` or `fn`
    |     help: `async` must come before `unsafe`: `async unsafe`
-LL | }
-   | - the item list ends here
```

Comment:
Ideally, I think it's best to leave this as it is.

Reply:
Hmm, this message indicates that it wasn't able to recover the rest of the item list if I understand right? I think this is an improvement as it's able to continue parsing other items in the list, so the message isn't really necessary